package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "express": "^4.18.2",
-    "nodemon": "^2.0.22"
+    "nodemon": "^2.0.22",
```

Comment:
Typically you'd put `nodemon` in `devDependencies`, using `npm install --save-dev nodemon`